### DISCUSSION TITLE
Add different color to Right-Click for menu cursor

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#15621] [Ideas] [answered]
Currently, Both M1 and M2 only changes the cursor's color to pink. Having both buttons change to the same color just feels inconsistent and just... not right. I think a different color would be better. Maybe add even more colors like M3 if it has some functionality.

Something like this:
![cursorsm1m2](https://user-images.githubusercontent.com/64581009/141684980-e649f3f9-fd03-4804-8bc7-55194498a812.png)

**Comment by peppy** [comment, marked as answer]
Didn't you already post this? https://github.com/ppy/osu/discussions/15240

**Reply by NotGumballer91** [reply]
Oops. I forgot to search discussions before posting
